app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughTopicListFragmentTest.kt

```diff
@@ -179,7 +179,7 @@ class WalkthroughTopicListFragmentTest {
   }
 
   @Test
-  fun testWalkthroughWelcomeFragment_recyclerViewIndex4_lessonThumbnailIsCorrect() {
+  fun testWalkthroughTopicListFragment_topicCard_lessonThumbnailIsCorrect() {
```

Comment:
@anandwana001 should i make this ` testWalkthroughTopicListFragment_topicCardSelected_lessonThumbnailIsCorrect()`

Reply:
that way we have to make all `topicCardSelected` which might get repetitive. Going with the current one.